Alignment/APEEstimation/test/Misalignments/createRandomlyMisalignedGeometry_cfg.py

```diff
@@ -91,33 +111,36 @@
 scenariolabel = str(options.myScenario)
 if sigma > 0:
     scenariolabel = scenariolabel+str(sigma)
-outputfilename = "geometry_"+str(scenariolabel)+"_from"+process.GlobalTag.globaltag._value.replace('::All','')+".db"
+outputfilename = "geometry_"+str(scenariolabel)+"__from_"
+if options.inputDB is None:
+    outputfilename += process.GlobalTag.globaltag.value().replace("::All","")+".db"
```

Comment:
we don't support condDBv1 in 81x anymore, the `replace("::All","")` statement could be dropped